pagopa/io-app
# feat(IT Wallet): [SIW-214] Add PID to wallet

## Short description
This PR adds the PID credential to the wallet section, along with a detail screen.
Most of the changes are tied to the PID and will be generalised in the future.

## List of changes proposed in this pull request
- `locales/en/index.yml` & `locales/it/index.yml`: Adds locales;
- `ts/features/it-wallet/components/ClaimsList.tsx`:  Adds the issuer name to the claim list unrecognized data section;
- `ts/features/it-wallet/navigation/ItwStackNavigator.tsx`: Adds the new details screen to the navigator;
- `ts/features/it-wallet/navigation/params.ts`: Adds the new details screen params;
- `ts/features/it-wallet/navigation/routes.ts`: Adds the new presentation routes;
- `ts/features/it-wallet/screens/ItwCredentialDetails.tsx`: Adds the VC details screen (currently only for PID); 
- `ts/features/it-wallet/screens/ItwHomeScreen.tsx`: Adds the PID card to the wallet section;
- `ts/features/it-wallet/store/reducers/itwCredentials.ts`: Adds a selector for VCs;

## How to test
Complete a PID issuing flow and check if the card gets renders in the `Wallet` section. Then try to press it to navigate to the details screen.
